.github/workflows/actions.yml

```diff
@@ -8,13 +8,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
-      - name: Set up Cloud SDK
-        uses: google-github-actions/setup-gcloud@master
+        uses: actions/checkout@v3
+      - name: 'Authenticate Google Cloud with Service Account'
+        uses: google-github-actions/auth@v1
         with:
-          project_id: ${{ secrets.GCP_PROJECT_ID }}
-          service_account_key: ${{ secrets.GCP_SA_KEY }}
-          export_default_credentials: true
+          credentials_json: ${{ secrets.GCP_SERVICE_ACCOUNT_JSON }}
```

Comment:
I guess `${{ secrets.GCP_SERVICE_ACCOUNT_JSON }}` is almost the same as `${{ secrets.GCP_SA_KEY }}` .
Can we use the same variable name?

Reply:
updated

Reply:
LGTM